lib/bolt/plan_creator.rb

```diff
@@ -36,8 +36,8 @@ def self.validate_input(project, plan_name)
       prefix, _, basename = segment_plan_name(plan_name)
 
       unless prefix == project.name
-        message = "First segment of plan name '#{plan_name}' must match project name '#{project.name}'. "\
-                  "Did you mean '#{project.name}::#{plan_name}'?"
+        message = "Incomplete plan name: Plan names consist of the project or module name "\
+          "followed by the specific plan. Did you mean '#{project.name}::#{plan_name}'?"
```

Comment:
"specific plan name"? "rest of the plan name"? Something sounds a bit odd about "specific plan". @hestonhoffman , any thoughts?

Reply:
Hmmm. I see the issue with the original suggestion. "Plan name" is repetitive. How about something like this:
"Incomplete plan name: A plan name must be prefixed with the name of the project or module. Did you mean..."

Reply:
Yeah that seems better - I wasn't sure what to call it either :P